pkg/asset/cluster/ovirt/ovirt.go

```diff
@@ -0,0 +1,17 @@
+// Package ovirt extracts ovirt metadata from install configurations.
+package ovirt
+
+import (
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/ovirt"
+)
+
+// Metadata converts an install configuration to ovirt metadata.
+func Metadata(config *types.InstallConfig) *ovirt.Metadata {
+	return &ovirt.Metadata{
+		URL:      config.Platform.Ovirt.URL,
+		Username: config.Platform.Ovirt.Username,
```

Comment:
You should remove username and password from the metadata. E.g. what if a different user destroys the cluster than created it?

It looks like you already updated destroy code to read the config and read in username and password.